src/main/scala/rocket/PTW.scala

```diff
@@ -16,25 +16,36 @@ import freechips.rocketchip.util._
 import freechips.rocketchip.util.property
 import scala.collection.mutable.ListBuffer
 
+/** PTE request from TLB to PTW
+  *
+  * TLB send a PTE request to PTW when TLB miss and it needs a page table walk
+  */
 class PTWReq(implicit p: Parameters) extends CoreBundle()(p) {
   val addr = UInt(width = vpnBits)
   val need_gpa = Bool()
   val vstage1 = Bool()
   val stage2 = Bool()
```

Comment:
what does these signals mean?
The interface signals should be well-documented:
from where, to where, the usage of this signal.

Reply:
Work in IO signals in hypervisor mode has been marked unfinished.